README.win32.adoc

```diff
@@ -29,7 +29,8 @@ Here is a summary of the main differences between these ports:
 (*):: Cygwin-generated `.exe` files are linked with the Cygwin DLL. Since
 version 2.5.2 released in June 2016, this DLL is licensed under the LGPLv3 with
 a static linking exception; prior versions of the DLL are distributed under the
-GPL. Thus, these `.exe` files can only be distributed under compatible license.
+GPL (the cygwin version can be obtained with the `uname -r` command). Thus,
+these `.exe` files can only be distributed under a compatible license.
```

Comment:
How does this look, therefore (it reads oddly if you try to remove the reference to the GPL completely):
```
(*):: Executables generated by the native GCC package in Cygwin are linked with
the Cygwin DLL and require this to be distributed with your programs. Cygwin 2.5.2
or higher is recommended (the Cygwin version can be obtained with `uname -r`) -
as earlier versions are licensed under the GPL and thus require compiled executables
to be redistributed under a compatible licence. Executables generated by Microsoft 
Visual C++ or the Mingw-w64 compilers (even when run in Cygwin as `i686-w64-mingw32-gcc`
or `x86_64-w64-mingw32-gcc`) are not linked against this DLL. 
 ```
The original sentence I think predates the introduction of the Mingw ports, and I do think it's worth the clarification that the mingw compilers (which are officially supposed to be the Cygwin-packaged ones) are not affected, even if you happen to be using Cygwin < 2.5.2